### ISSUE TITLE
Label of button function from addon-knobs unnecessary suffix based on groupId

### ISSUE THREAD

**Issue by kalinchernev** [storybookjs/storybook#10766]
# Question

When using `button` from `@storybook/addon-knobs` in the documented way:

```js
storiesOf('string', module)
  .addDecorator(withKnobs)
  .addDecorator(withNotes)
  .addDecorator(withCode)
  .add(
    'default',
    () => {
      button('string', 'handler', 'string');
    },
    {
      notes: { markdown: notes, json: data },
    }
  )
```

Button labels get undesired suffices based on `groupId`: 

![storybook_buttons_issue](https://user-images.githubusercontent.com/1923476/81820571-12d35f80-9531-11ea-81ad-c663ce52ebbf.png)

### Steps to reproduce

I believe that the behavior can be reproduced with any data as I've tried to change input parameters and having a groupId always yields the same behavior.

### Please specify which version of Storybook and optionally any affected addons that you're running

```
    "@storybook/addon-knobs": "5.3.18",
    "@storybook/addon-options": "5.3.18",
    "@storybook/addon-viewport": "5.3.18",
    "@storybook/html": "5.3.18",
```

### Thoughts

I've tracked that these are potentially leading the current behavior:
- [button knob function](https://github.com/storybookjs/storybook/blob/7f714d60b7dca4225239f34808c0ed07d2f2489d/addons/knobs/src/index.ts#L105)
- [knob name is using group id](https://github.com/storybookjs/storybook/blob/7f714d60b7dca4225239f34808c0ed07d2f2489d/addons/knobs/src/KnobManager.ts#L78)

It might be that this behavior is desired for storybook. However, having the suffix repetition in the UI looks a bit off having the group id placed on the tab already.

**Comment by shilman** [comment]
Hi gang, We’ve just released addon-controls in 6.0-beta!

Controls are portable, auto-generated knobs that are intended to replace addon-knobs long term.

- [Controls README](https://github.com/storybookjs/storybook/tree/next/addons/controls)
- [Migration instructions](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)

Please upgrade and try them out today. Thanks for your help and support getting this stable for release!

**Comment by shilman** [comment]
For anybody who is interested in Controls but don't know where to start, I've created a quick & dirty step-by-step walkthrough to go from a fresh CRA project to a working demo. Check it out:

=> [Storybook Controls w/ CRA & TypeScript](https://gist.github.com/shilman/69c1dd41a466bae137cc88bd2c6ef487)

There are also some "knobs to controls" migration docs in the Controls README:

=> [How do I migrate from addon-knobs?](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)
